rest/data/host_create.go

```diff
@@ -151,6 +151,64 @@ func createHostFromCommand(cmd model.PluginCommandConf) (*apimodels.CreateHost,
 }
 
 func (dc *DBCreateHostConnector) MakeIntentHost(taskID, userID, publicKey string, createHost apimodels.CreateHost) (*host.Host, error) {
+	if createHost.CloudProvider == evergreen.ProviderNameDocker {
+		return dc.makeDockerIntentHost(userID, createHost)
+	}
+	return dc.makeEC2IntentHost(taskID, userID, publicKey, createHost)
+}
+
+func (dc *DBCreateHostConnector) makeDockerIntentHost(userID string, createHost apimodels.CreateHost) (*host.Host, error) {
```

Comment:
This should be tested (see route/host_create_test.go, `TestMakeIntentHost`), though maybe not to that extreme.

Reply:
I did add a TestHostCreateDocker to this file already, let me know if you're referencing a different kind of testing.

Reply:
Ah, I misread that test as testing something else.